host_applications/linux/apps/raspicam/RaspiCamControl.c

```diff
@@ -1811,6 +1813,22 @@ void default_camera_control_callback(MMAL_PORT_T *port, MMAL_BUFFER_HEADER_T *bu
                         settings->awb_red_gain.num, settings->awb_red_gain.den,
                         settings->awb_blue_gain.num, settings->awb_blue_gain.den);
       }
+      case MMAL_PARAMETER_CAPTURE_STATUS:
+      {
+         wiringPiSetup();
+         pinMode(21, OUTPUT);
```

Comment:
One interesting thing I could test is whether we could control the pin mode (output/input) in the python code instead of here. We then might be able to hackily control whether the led actually gets turned on/off.

Reply:
seems like it'd also be easy to check some kind of file/socket/whatever.
should we at least set the pin number to a variable or a `#DEFINE` ? :grimacing:

Reply:
doesn't seem worth the time to me